vdp/connector/v1alpha/connector.proto

```diff
@@ -123,6 +123,9 @@ message Connector {
       [ (google.api.field_behavior) = OUTPUT_ONLY ];
   // Connector visibility including public or private
   Visibility visibility = 15 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  // Embed the content of the connector_definition_detail
+  ConnectorDefinition connector_definition_detail = 16
```

Comment:
@donch1989, this is a good change. 
But if we provide `connector_definition_detail`, the original `connector_definition` (see below) is a bit redundant. 

```
// ConnectorDefinition resource
  string connector_definition = 4 [
    (google.api.field_behavior) = IMMUTABLE,
    (google.api.resource_reference) = {
      type : "api.instill.tech/ConnectorDefinition"
    }
  ];
 ```
   
Should we remove the original `connector_definition` and rename `connector_definition_detail` as `connector_definition`?

Reply:
Yeah, that can be a choice. 

@EiffelFly Is that ok if we change to this:
- VIEW_BASIC: connector_definition will be null
- VIEW_FULL: connector_definition will be the full content of connector_definition

Reply:
How about 

- VIEW_BASIC: connector_definition will be connector_definition title
- VIEW_FULL: connector_definition will be the full content of connector_definition

Because I won't need the full connector_definition but the title of it sometimes

Reply:
Did you mean for the VIEW_BASIC, that field will be string?

Reply:
yes!

Reply:
@donch1989 and @EiffelFly, the [Google AIP-157](https://google.aip.dev/157) practise mentioned that typically, the VIEW is used to return a partial response, not changing the type of a field in the response.

I would suggest we don't break the rule.

Reply:
I don't like the same field will have different format when VIEW is different. Let me think.

Reply:
ok, how about this?
original `connector_definition` is renamed to `connector_definition_id` (just to check it is the ID of a connector definition, right?)

and new `connector_definition_detail` -> `connetor_definition`, which is output only and only include in the full view

Reply:
Yes, that will be good. 
But I think it should be `connector_definition_name` because the content is `connector-definitions/xxx` not `xxx`

Reply:
> I don't like the same field will have different format when VIEW is different. Let me think.

You are right, we should no do that!

Reply:
updated.